ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -151,6 +157,9 @@ export default React.memo<Props>(function VideoJs(props: Props) {
     ],
     autoplay: false,
     poster: poster, // thumb looks bad in app, and if autoplay, flashing poster is annoying
+    // @if TARGET='web'
+    techOrder: ['chromecast', 'html5'],
```

Comment:
What does `html5` mean here?

Reply:
It's the default "tech" for videojs, the one that wraps `<video>` I guess.  Other potential ones include Flash and Silverlight.
Per plugin documentation, it wants to be before the default tech.